tests/integration/test_case.go

```diff
@@ -246,6 +246,7 @@ type CreateIndex struct {
 	// The names of the fields to index. Used only for composite indexes.
 	FieldsNames []string
 	// The directions of the 'FieldsNames' to index. Used only for composite indexes.
+	// If not provided all fields will be indexed in ascending order.
 	Directions []client.IndexDirection
```

Comment:
suggestion: It feels a bit clunky to have to separate fields from directions like this.  Happy for it to be done outside of this PR but I do think it would be nicer to have something like the below:
```
Fields []client.IndexedFieldDescription
```

Reply:
I don't think I agree. This will make every such test action more wordy. There will be a few tests that test different directions.

I just added a comment there. You were not supposed to comment on this part :D 

The next task I'm working on involves different directions. I'd suggest we check how it's going to look in those tests and discuss it there.

Reply:
> I just added a comment there. You were not supposed to comment on this part :D

:grin: :grin: 

> The next task I'm working on involves different directions. I'd suggest we check how it's going to look in those tests and discuss it there.

That sounds sensible, I'm happy with that :)